src/runtime/crt/common/crt_runtime_api.c

```diff
@@ -526,6 +526,11 @@ tvm_crt_error_t RunTimeEvaluator(tvm_function_index_t function_index, TVMValue*
     int exec_count = 0;
     // do-while structure ensures we run even when `min_repeat_ms` isn't set (i.e., is 0).
     do {
+      err = TVMPlatformBeforeMeasurement();
+      if (err != kTvmErrorNoError && err != kTvmErrorFunctionCallNotImplemented) {
```

Comment:
i think you could just do the one check `err != kTvmErrorNoError` and return `kTvmErrorNoError` by default from the weak-linked functions. thoughts?

Reply:
Yes, that would also work.

Reply:
Just committed the change.